gen_manifest/gen_manifest.py

```diff
@@ -251,9 +248,13 @@ def get_region_tag(sample_file_path):
   return region_tags[0]
 
 
-def glob_non_yaml(pattern):
-  '''Recursively globs for `pattern`, ignoring "*.yaml" and ".yml" files.'''
-  matches = glob(pattern, recursive=True)
+def glob_non_yaml(all_patterns):
+  '''Recursively globs for each pattern in `all_patterns`, ignoring "*.yaml" and ".yml" files.'''
+  matches = []
+  for pattern in all_patterns:
+      matches.extend(glob(pattern, recursive=True))
+  matches = list(set(matches)) # eliminate duplicates
+  matches.sort()               # want deterministic order for tests
```

Comment:
Nit: can rewrite as a single expression
```python
matches = sorted(set(glob(pattern, recursive=True) for pattern in all_patterns))  # Deterministically ordered unique patterns.
```

Reply:
It would be really cool if `sort` returned a reference to the list being sorted, because then it's easy to use it in compact expressions.